src/views/RecordingsView.vue

```diff
@@ -461,7 +461,7 @@ export default {
             }
             return str;
           };
-          const formatDate = date =>
+          const formatQueryDate = date =>
```

Comment:
should this be it's own method instead of being defined inline?

Reply:
Also could just use a library to help format the date https://www.npmjs.com/package/dateformat